nncf/common/quantization/quantizer_propagation/solver.py

```diff
@@ -333,6 +333,7 @@ def __init__(
         post_processing_marker_metatypes: Optional[list[type[OperatorMetatype]]] = None,
         metatypes_to_ignore: Optional[list[type[OperatorMetatype]]] = None,
         scales_unification_map: Optional[dict[type[OperatorMetatype], list[type[OperatorMetatype]]]] = None,
+        is_fp8: bool = False,
```

Comment:
@andreyanufr, @AlexanderDokuchaev, please provide suggestion how to avoid passing is_fp8 parameter in the solver.

Reply:
Tried to change QuantizationConfig to check dtype, but it requires to many changes in work of solver and hwconfigs,
So is_fp8 parameters is good solution now.